mapper/mapper.go

```diff
@@ -46,16 +46,17 @@ type ErrorModel struct {
 
 type AreaModel struct {
 	coreModel.Page
-	Name      string           `json:"name"`
-	Code      string           `json:"code"`
-	Level     string           `json:"level"`
-	AreaType  string           `json:"area_type"`
-	NomisLink string           `json:"nomis_link"`
-	Ancestors []areas.Ancestor `json:"ancestors"`
-	Siblings  []AreaModel      `json:"siblings"`
-	Children  []AreaModel      `json:"children"`
-	Relations []RelationLink   `json:"relations"`
-	Version   string           `json:"area_profiles_version"`
+	Name                string           `json:"name"`
+	Code                string           `json:"code"`
+	Level               string           `json:"level"`
+	AreaType            string           `json:"area_type"`
+	NomisLink           string           `json:"nomis_link"`
+	Ancestors           []areas.Ancestor `json:"ancestors"`
+	Siblings            []AreaModel      `json:"siblings"`
+	Children            []AreaModel      `json:"children"`
+	Relations           []RelationLink   `json:"relations"`
+	Version             string           `json:"area_profiles_version"`
```

Comment:
Should this be AreaProfilesVersion?

Reply:
There's only one call to a 'version' value so simpler in the view to refer to it as a version...